File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        String s= ""+n;
        int[] answer = new int[s.length()];
        long m=n;
        for (int i=0; i<s.length(); i++){
            answer[i] = (int) (m % 10);
            m = m / 10;
        }
        return answer;
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.kt
class Solution {
    fun solution(n: Long): IntArray {
        var answer = intArrayOf()
        var list=ArrayList<Long>()
        var m:Long=n
        while(m != 0L){
            list.add(m % 10L)
            m=m/10L
        }
        answer = list.map { it.toInt() }.toIntArray()
        return answer
    }
}